app/javascript/controllers/file_auth_controller.js

```diff
@@ -8,7 +8,7 @@ export default class extends Controller {
 
   addPostCallbackListener() {
     window.addEventListener("message", (event) => {
-      this.iframe.remove()
+      if (this.iframe) this.iframe.remove()
```

Comment:
Where would the callback come from if not in the iframe?

Reply:
This is really for locally development. If you go to the iframe URL it gives this error.

Reply:
can you show me? The iframe I'm referring to is the one created here: https://github.com/sul-dlss/sul-embed/blob/main/app/javascript/controllers/file_auth_controller.js#L206